web_programming/convert_number_to_words.py

```diff
@@ -1,106 +1,83 @@
 import math
 
 
-def convert(number: int) -> str:
-    """
-    Given a number return the number in words.
+def int_to_en(num):
```

Comment:
https://beta.ruff.rs/docs/rules/too-many-return-statements is a code complexity rule that we are not going to be willing to break.